src/types/account.ts

```diff
@@ -214,6 +215,8 @@ export type Account = {
   swapHistory: SwapOperation[];
   // Hash used to discard tx history on sync
   syncHash?: string;
+  // Array of NFTs computed by diffing NFTOperations
+  NFT?: NFT[];
```

Comment:
would you be so kind and rename this in camel case for the variable :grin: 
you can keep use pascal case for types too

Reply:
`nfts`